PageObjects/ReviewYourAnswersPage.js

```diff
@@ -4,11 +4,34 @@ class ReviewYourAnswersPage {
 
     constructor(page) {
         this.header = page.locator("(//div[text()=' Review Your Answers '])[2]");
+        this.preAppPageQuesEditBtn = page.locator("//div[@btnname='summaryBtn1']//button[1]");

```

Comment:
Please update the locator to closest and its a bad practice to use div, instead get by role or some other method.

Reply:
I can update the locators to closest and for this dev team needs to add name, test id's to the elements to inspect them. Thanks.